R/db-interface.R

```diff
@@ -1,28 +1,28 @@
-#' Copy a `dm`-object to a `src`/`con`
+#' Copy a `dm`-object to a `src` / `con`
```

Comment:
Further below I'm using three synonymous explanations that work in this context.

```suggestion
#' Copy a `dm`-object to a different data source
```

Reply:
I think that the problem here is that 'src' and 'con' are dplyr objects that may be unknown to people who will use dm without having used dplyr before. So these objects would need to be defined -- what kind of objects are they and what is the difference between them? Local databases, remote databases, spreadsheets, csv files,  data frames?